docs/user-guide/nginx-configuration/annotations-risk.md

```diff
@@ -105,6 +105,7 @@
 | Redirect | permanent-redirect-code | Low | location |
 | Redirect | temporal-redirect | Medium | location |
 | Redirect | temporal-redirect-code | Low | location |
+| RelativeRedirects | relative-redirects | Low | location |
```

Comment:
Lets keep it in the redirect group and not make a new one.

Reply:
Ok